vme/zone/basis.zon

```diff
@@ -614,6 +614,7 @@ code
       informer();
       login_modify(self);
       exec("look", self);
+      exec("cupdate", self);
```

Comment:
We might want to change this to calling a DIL function rather than executing a command. I'll have to check out what this does to the login process.